Formula/fdroidcl.rb

```diff
@@ -0,0 +1,24 @@
+class Fdroidcl < Formula
+  desc "F-Droid desktop client"
+  homepage "https://github.com/mvdan/fdroidcl"
+  url "https://github.com/mvdan/fdroidcl/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "532a8c4c93216cbf13378ff409c06a08d48e8baee6119a50ed43dc0ce9ec7879"
+  license "BSD-3-Clause"
+  head "https://github.com/mvdan/fdroidcl.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+    bin.install "fdroidcl"
```

Comment:
This line is causing the current build failure and should be removed. The `go build` step with `*std_go_args` will automatically install it into the prefix. The binary does not need to be installed separately (the binary doesn't exist in the buildpath anymore which is why the error is occurring).

Reply:
Ah, thank you, that makes sense. I'll push a commit soon.